From ba8219e0ca0bde2a3635a589659d8840961dba7b Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Thu, 19 Oct 2017 11:10:23 +1300
Subject: Add basic Makefile.PL and Tests

This distribution is so old, it could possibly have been authored
before EUMM and friends were common practice.

Bug: https://rt.cpan.org/Ticket/Display.html?id=13921
---
 Makefile.PL | 15 +++++++++++++++
 t/compile.t | 12 ++++++++++++
 2 files changed, 27 insertions(+)
 create mode 100644 Makefile.PL
 create mode 100644 t/compile.t

diff --git a/Makefile.PL b/Makefile.PL
new file mode 100644
index 0000000..f272d63
--- /dev/null
+++ b/Makefile.PL
@@ -0,0 +1,15 @@
+use ExtUtils::MakeMaker;
+
+WriteMakefile(
+    NAME          => 'ConfigReader',
+    VERSION_FROM  => "DirectiveStyle.pm",
+    ABSTRACT_FROM => "ConfigReader.pod",
+    LICENSE       => "gpl",
+    PREREQ_PM     => {},
+    PM            => {
+        'ConfigReader.pod'  => '$(INST_LIB)/ConfigReader.pod',
+        'DirectiveStyle.pm' => '$(INST_LIB)/ConfigReader/DirectiveStyle.pm',
+        'Spec.pm'           => '$(INST_LIB)/ConfigReader/Spec.pm',
+        'Values.pm'         => '$(INST_LIB)/ConfigReader/Values.pm',
+    },
+);
diff --git a/t/compile.t b/t/compile.t
new file mode 100644
index 0000000..46c1f1d
--- /dev/null
+++ b/t/compile.t
@@ -0,0 +1,12 @@
+use strict;
+use warnings;
+
+use Test::More;
+
+require_ok("ConfigReader::DirectiveStyle");
+require_ok("ConfigReader::Spec");
+require_ok("ConfigReader::Values");
+
+done_testing;
+
+
-- 
2.14.2

